workspaces/orchestrator/docs/Contributors.md

```diff
@@ -0,0 +1,70 @@
+## How to use the Git Hub identity provider
+
+The [app config](../app-config.yaml) and the [App.tsx](../packages/app/src/App.tsx) files contain sufficient configuration to use the Git Hub identity provider.
+
+Follow these steps to login to backstage using your github account:
+
+### Create a Github OAuth App
+
+Go to your Github account -> Setting -> Developer settings -> OAuth Apps -> New OAuth App.
+Enter a name, enter http://localhost:3000 to Homepage URL and http://localhost:7007/api/auth/github/handler/frame to Authorization callback URL and click on Register applicatdion.
+
+### Update backstage user entity
+
+Update [users.yaml](../users.yaml) to match you Github account. Put your github username in the metadata.name field, and you github email in spec.profile.email field.
+
+### Setup environment variables and run
+
+```
+export AUTH_GITHUB_CLIENT_ID=...fill from OAuth App client ID
+export AUTH_GITHUB_CLIENT_SECRET=...fill from OAuth App client secret
+
+yarn dev
+```
+
+## API Development instruction
```

Comment:
Shouldn't this section go under orchestrator-common/README.md and be pointed from here or the other way around?